pySDC/core/Sweeper.py

```diff
@@ -270,40 +280,96 @@ def rho(x):
             self.parallelizable = True
 
         elif qd_type == "MIN-SR-S":
+
             M = QDmat.shape[0] - 1
-            Q = coll.Qmat[1:, 1:]
-            nodes = coll.nodes
+            quadType = coll.quad_type
+            nodeType = coll.node_type
+
+            # Main function to compute coefficients
+            def computeCoeffs(m, a=None, b=None):
```

Comment:
the function signature is very nondescript. Without understanding the entire thing, I have no indication what `a` and `b` might be. In fact, as I go from the top, I don't get it.

Reply:
More documentation have been added